int16 size
diagnostic_msgs/KeyValue[] settings
